components/dropbox/sources/new-file/new-file.mjs

```diff
@@ -36,11 +48,20 @@ module.exports = {
       this.db.set("last_file_mod_time", startTime);
     },
   },
-  async run(event) {
+  methods: {
+    getMeta(id, summary) {
+      return {
+        id,
+        summary,
+        tz: Date.now(),
+      };
+    },
+  },
```

Comment:
If you create a `common.mjs` file in sources and include `...common.methods` here, you can remove the `getMeta()` method. Anytime we're accessing `db`, we should use getter & setter methods. I'd suggest adding methods `_getLastFileModTime()` and `_setLastFileModTime(lastFileModTime)` here and using them instead of directly accessing `this.db`.